lib/beaker/host/pswindows/pkg.rb

```diff
@@ -7,15 +7,21 @@ def check_for_command(name)
   end
 
   def check_for_package(name)
```

Comment:
This can be implemented with the following powershell:

```PowerShell
Get-ItemProperty HKLM:\Software\Microsoft\Windows\CurrentVersion\Uninstall\* | Select-Object DisplayName | Where-Object { $_.DisplayName -eq "Microsoft .NET Framework 4.5.2" } | % { $_.DisplayName }
```